diff --git a/node_modules/expo-asset/android/build.gradle b/node_modules/expo-asset/android/build.gradle
index 4ddc82a..06784dc 100644
--- a/node_modules/expo-asset/android/build.gradle
+++ b/node_modules/expo-asset/android/build.gradle
@@ -1,8 +1,13 @@
 plugins {
   id 'com.android.library'
-  id 'expo-module-gradle-plugin'
 }
 
+def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
+apply from: expoModulesCorePlugin
+applyKotlinExpoModulesCorePlugin()
+useDefaultAndroidSdkVersions()
+useCoreDependencies()
+
 group = 'expo.modules.asset'
 version = '55.0.14'
 
